games-strategy/opendune/opendune-0.9.recipe

```diff
@@ -10,23 +10,22 @@ for the game to run properly."
 HOMEPAGE="http://www.opendune.org/"
 COPYRIGHT="2009-2018 OpenDUNE Developers"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="https://github.com/OpenDUNE/OpenDUNE/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="8d5ed67669df1f17a44c097d9b6bee4e0623ce2a37f11938ce9cd77de546d06c"
 SOURCE_FILENAME="OpenDUNE-$portVersion.tar.gz"
 SOURCE_DIR="OpenDUNE-$portVersion"
 PATCHES="opendune-$portVersion.patchset"
 ADDITIONAL_FILES="opendune.rdef"
 
-ARCHITECTURES="all !x86_gcc2 ?x86"
+ARCHITECTURES="all !x86_gcc2 x86 x86_64"
```

Comment:
The non-marked individual architectures are redundant with `all`. Also, there is no `x86` primary architecture.

```suggestion
ARCHITECTURES="all !x86_gcc2"
```